### PR TITLE
Update version: glueckkanja.KONNEKT version 2.10.1

### PR DESCRIPTION
**Pull request by damn-good-b0t**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/193899)

### CODE REVIEW

File: manifests/g/glueckkanja/KONNEKT/2.10.1/glueckkanja.KONNEKT.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://trial.konnekt.io/releases/Konnekt-X86-2.10.1.0.Msi
+  InstallerSha256: 6AC85CC3B848EE3882C25ACCC2414FE0B85D28DC1D02F1AEC4231A1B681E1D11
+  ProductCode: '{CB6E858C-4926-4B72-9437-23A78346D9DB}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Konnekt
+    Publisher: glueckkanja-gab AG
+    DisplayVersion: 2.10.1.0
```
Comment (Trenly) [inline manifests/g/glueckkanja/KONNEKT/2.10.1/glueckkanja.KONNEKT.installer.yaml:18]:
```suggestion
```

File: manifests/g/glueckkanja/KONNEKT/2.10.1/glueckkanja.KONNEKT.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    ProductCode: '{CB6E858C-4926-4B72-9437-23A78346D9DB}'
+    UpgradeCode: '{32C9E0B0-DFBE-4307-BF29-6E16FAD45F80}'
+- Architecture: x64
+  InstallerUrl: https://trial.konnekt.io/releases/Konnekt-X64-2.10.1.0.Msi
+  InstallerSha256: 6CD51A9489F7E495C602947CB08DC5D87552DD25EFEA172CFB105BD41324C094
+  ProductCode: '{04B2655B-5735-47C9-96FD-D6FDFF8E8CD7}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Konnekt
+    Publisher: glueckkanja-gab AG
+    DisplayVersion: 2.10.1.0
```
Comment (Trenly) [inline manifests/g/glueckkanja/KONNEKT/2.10.1/glueckkanja.KONNEKT.installer.yaml:28]:
```suggestion
```

File: manifests/g/glueckkanja/KONNEKT/2.10.1/glueckkanja.KONNEKT.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    ProductCode: '{04B2655B-5735-47C9-96FD-D6FDFF8E8CD7}'
+    UpgradeCode: '{32C9E0B0-DFBE-4307-BF29-6E16FAD45F80}'
+- Architecture: arm64
+  InstallerUrl: https://trial.konnekt.io/releases/Konnekt-Arm64-2.10.1.0.Msi
+  InstallerSha256: 7170F77F91FC1BACCC2A155957F3D874C687561B6C248B8EFA546151424F2F55
+  ProductCode: '{BDB35071-4C6D-4FC1-B735-5A26F4CD9D8B}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Konnekt
+    Publisher: glueckkanja-gab AG
+    DisplayVersion: 2.10.1.0
```
Comment (Trenly) [inline manifests/g/glueckkanja/KONNEKT/2.10.1/glueckkanja.KONNEKT.installer.yaml:38]:
```suggestion
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Trailing `.0` is already considered in version comparison, no need for DisplayVersion for this package

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-11-27 09:45:08.249 [CLI ] Package hash verification failed. SHA256 in manifest [d665af230cda379924982e1d13841f7b2efe7cf75a3ebce821f89dece335a93c] does not match download [183b008a19013117e51c5839a9369584dec268749ba61505e6fb022a15d9987c] with file size [6037504] and content type []

(Automated response - build 899.)

**Pull request merged**